package.json

```diff
@@ -175,7 +176,8 @@
     "testEnvironment": "jsdom",
     "testURL": "http://localhost",
     "moduleNameMapper": {
-      "./src(.*)$": "<rootDir>/src/$1"
+      "./src(.*)$": "<rootDir>/src/$1",
+      "\\.(css|scss)$": "<rootDir>/node_modules/@reach/tooltip/index.js"
```

Comment:
Heya @MayaUribe! 🤗I'm curious about this: shouldn't this be matched against the `styles.css` file? 🤔